### PR TITLE
resolvedEdits

### PR DESCRIPTION
**Pull request by gcater**
Took a second to navigate git, but have taken out react-query and the book interface. Same things as before so amended prisma, set google login, and setup a basic book TRPC mutation and query. Will soon change how this works. 

Screenshot 2024-03-26 at 7 16 44 PM

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -128,7 +122,7 @@ function bookDisplay() {
 
   return (
     <div>
-      {data.map((book: Book) => (
+      {data.map((book: any) => (
```
Comment (Evanc123) [inline src/pages/index.tsx]: so this shouldn't be an Any (we should almost never use any), it should come from Book exported by the `trpc` client

**Review by Evanc123 — CHANGES_REQUESTED** [review]

File: src/pages/index.tsx (hunk 1 above)
Comment (Evanc123) [reply src/pages/index.tsx]: plz also fix the merge conflicts

Comment (Evanc123) [reply src/pages/index.tsx]: thanks for the photo! helpful

**Comment by gcater** [comment]
Fixed some git errors and took the any out

**Review by Evanc123 — APPROVED** [review]

**Pull request merged**